src/plugins/github/parseMarkdown.test.js

```diff
@@ -50,6 +50,54 @@ describe("plugins/github/parseMarkdown", () => {
       ];
       expect(textBlocks(input)).toEqual(expected);
     });
+
+    it("includes text inside of non-code HTML elements", () => {
+      const input = "My <strong>#1</strong> pal";
+      expect(textBlocks(input)).toEqual(["My #1 pal"]);
+    });
+
+    it('strips HTML "code" elements', () => {
+      const input = "My <code>#1</code> pal";
+      expect(textBlocks(input)).toEqual(["My  pal"]);
+    });
+
+    it('strips subtrees rooted at HTML "code" elements', () => {
+      const input = "My <code>#1 <strong>*and* #2</strong></code> pals";
+      expect(textBlocks(input)).toEqual(["My  pals"]);
+    });
+
+    it('strips "code" elements within "code" elements', () => {
+      const input = "see <code>#1 and <code>#2</code> okay</code>";
+      expect(textBlocks(input)).toEqual(["see "]);
+    });
+
+    it('handles comments and CDATA within "code" elements', () => {
+      // These are "html_inline" nodes, but are not HTML elements. They
+      // may contain closing-tag sequences, but these do not actually
+      // close a tag.
+      const input = [
+        "note",
+        "<code>alpha ",
+        "<!-- bravo </code> charlie --> ",
+        "<![CDATA[delta </code> echo]]> ",
+        "foxtrot</code> ",
+        "well",
+      ].join("");
+      expect(textBlocks(input)).toEqual(["note well"]);
+    });
+
+    it('strips HTML "pre" blocks and subtrees', () => {
+      // "pre" is not handled specially; all blocks are skipped.
+      const input =
+        "Hello\n\n<pre>some pre-formatted <code>code</code></pre>\n\nworld";
+      expect(textBlocks(input)).toEqual(["Hello", "world"]);
+    });
+
+    it('strips non-"pre" blocks and subtrees', () => {
```

Comment:
This is a little weird, but I'm OK merging this for now.

Reply:
Agreed that it’s a bit weird that we ignore `div`s and `details`s and
such. It bears noting that this is the existing behavior. In particular,
CommonMark treats anything inside a block-level HTML element as raw. For
instance, [`<div>a **b** c</div>`][dingus] renders as a div containing
some literal asterisks and no `strong`s.

So, to make this _not_ be the case, we’d need to recursively seek out
these HTML blocks and re-run Markdown parsing on their contents. We
could consider that as a feature request, but it’s certainly outside the
purview of this commit.

[dingus]: https://spec.commonmark.org/dingus/?text=%3Cdiv%3Ea%20**b**%20c%3C%2Fdiv%3E

Reply:
Note this deviates from GitHub's reference detection.

<div>
**#907**
</div>

Observe that this created a reference to this pull.

Reply:
Yep, this is a good point.